pyproject.toml

```
@@ -9,7 +9,7 @@ author = "yaq developers"
 home-page = "https://yaq.fyi"
 description-file = "README.md"
 requires-python = ">=3.7"
-requires = ["yaqd-core>=2021.03.0", "setuptools", "cffi>=1.13", "numpy>=1.20"]
+requires = ["yaqd-core>=2021.03.0", "setuptools", "cffi>=1.13", "numpy>=1.20", "yaqc"]
```

```suggestion
requires = ["yaqd-core>=2021.03.0", "setuptools", "cffi>=1.13", "numpy>=1.20"]
```
we only want to list dependencies for the daemon itself; since yaqc is for client testing, we should not include it here.

---

It is being used for getting the position of the monochromator,  see lines 300-309 of _andorsdk2_ixon.py .

---

ah yes.  my mistake